pkg/transformer/aggr_loader.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package transformer
+
+import "github.com/arana-db/parser/ast"
+
+type AggrLoader struct {
+	Aggrs [][]string
+	Alias []string
+	Name  []string
+}
+
+func (a *AggrLoader) Enter(n ast.Node) (node ast.Node, skipChildren bool) {
+	fieldAggr := make([]string, 0, 10)
+	var enter func(n ast.Node) (node ast.Node, skipChildren bool)
+	enter = func(n ast.Node) (node ast.Node, skipChildren bool) {
```

Comment:
Moving it to a standalone method of `AggrLoader` will be better, I think.

Reply:
addressed